src/app/content/components/Page/contentLinkHandler.ts

```diff
@@ -88,7 +88,7 @@ export const contentLinkHandler = (anchor: HTMLAnchorElement, getProps: () => Co
       return;
     }
 
-    const base = new URL(assertWindow().location);
+    const base = new URL(assertWindow().location.href);
```

Comment:
this seems bad

Reply:
the `URL` accepts `string` so without it there is an error `Argument of type 'Location' is not assignable to parameter of type 'string'.ts(2345)`
`location.href` contains the whole URL so I think it is okay, because we are reseting `href` and `search` below anyway